components/support/rs_client/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "rs_client"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
we should add at least the `license`, `description` and `author` fields here: https://github.com/mozilla/application-services/blob/bd5ff1a29747702cbc95be998903be206c24506a/components/nimbus/Cargo.toml#L4-L9

Reply:
The question of authorship is an interesting one. I don't want to volunteer the Nimbus team, but I'm also not sure signing the Android mobile team up for it makes a ton of sense either in case of lost Rust context. I can check in with my team, but I'd be interested in any suggestions here as well

Reply:
We can add multiple so  Nimbus team  + Android team seems sensible - Nimbus wrote the pre-existing bits and you are moving down/evolving it!

We don't use the `Cargo.toml` as the absolute source of truth for ownership, but it's a good reference for who wrote the code!